src/main/java/com/fortanix/sdkms/performance/sampler/jce/EncryptDecryptBaseJCESampler.java

```diff
@@ -26,13 +31,17 @@
     protected Cipher cipher;
     protected AlgorithmParameters params;
     protected String input;
+    protected KeyPair keyPair;
 
     @Override
     public void setupTest(JavaSamplerContext context) {
         super.setupTest(context);
         String algorithm = context.getParameter(ALGORITHM, "AES");
         int keySize = context.getIntParameter(KEY_SIZE, 128);
         this.mode = CryptMode.fromValue(context.getParameter(MODE, "CBC"));
+        if(algorithm.equals("RSA")) {
+            this.mode = CryptMode.fromValue(context.getParameter(MODE, "OAEP_MGF1_SHA1"));
+        }
```

Comment:
is hard coding mode "OAEP_MGF1_SHA1" consciously taken decision or is it just random we chose?